app/src/Routes/Works/works.json

```diff
@@ -4,16 +4,49 @@
       {
         "title": "Dùn Èideann Blogh",
         "type": "double concerto",
+        "description": "Composed in 2006, written for and dedicated to Robert Walzel, clarinet.  Premiered in the summer of 2007 by the Utah Arts Festival Orchestra, Salt Lake City, with Robert Walzel, clarinet, and Lori Wike, bassoon, Andrew Rindfleisch conductor.",
+        "attribution": "Recorded by The Moravian Philharmonic Orchestra, Robert Walzel, clarinet, Eric Stomberg, bassoon, on <i>Cross Connection</i>; Navona Records, available at <a href=\"https://www.amazon.com/Wolking-Balentine-Connection-Various-artists/dp/B00BLWHTGO\" target=\"_blank\">Amazon</a> and <a href=\"https://itunes.apple.com/us/album/wolking-balentine-cross-connection/id608515694\" target=\"blank\"> iTunes.</a>",
         "instruments": [
-          "clarinet",
-          "bassoon",
-          "orchestra"
+          "Solo Clarinet",
+          "Solo Bassoon",
+          "Single WW: Flute, Oboe, Clarinet, bassoon",
+          "Single Brass: Horn, Trumpet, Trombone",
+          "2 Percussion",
+          "Piano",
+          "Strings"
         ],
-        "detailLink": "TBD"
+        "movements": [
```

Comment:
hrm.. a bunch of these attributes aren't taken into account on the page. i'll need to make some changes

Reply:
yeah none of these are for the works page, we'll need some child page routing to display them